### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by astridx** [maplibre/maplibre-gl-js-docs#51] [Q&A]
I use the plugin [mapbox-gl-compare](https://github.com/mapbox/mapbox-gl-compare). To use it together with MapLibre, I had to change one line.

I have seen that other plugins have been taken over to https://github.com/maplibre.

Now I don't know what the desired process is. Would you like to have my modification? You can find it here: https://github.com/astridx/maplibre-gl-compare

**Comment by wipfli** [comment]
Thanks for this. Would you volunteer to be the maintainer of something like https://github.com/maplibre/maplibre-gl-compare? We can definitely do this if you or someone else will feel responsible for the repo 😄

**Comment by wipfli** [comment]
The other option is to update the plugins list to point to your repo:

https://github.com/maplibre/maplibre-gl-js-docs/blob/58bafb7da47e359e0810495383cadac3a7fe528f/docs/data/plugins.json#L11-L14

Feel free to edit this and make a pull request

**Reply by wipfli** [reply]
This is the source code for https://maplibre.org/maplibre-gl-js-docs/plugins/

**Comment by astridx** [comment]
I would prefer all plugins to be together, so a address like
https://github.com/maplibre/maplibre-gl-compare

If you like, I would maintan as best I can at that place.

What would you prefer?

**Reply by wipfli** [reply]
Sounds good to me. Thanks for helping with maplibre :)

**Reply by wipfli** [reply]
Please open a pull request with your code on https://github.com/maplibre/maplibre-gl-compare
